remoteappmanager/static/js/home/views/application_list_view.js

```diff
@@ -3,10 +3,63 @@ define([
 ], function (Vue) {
     'use strict';
 
-    /* Create application_list ViewModel
-    (will next be wrapped in a main ViewModel which will contain the
-    applicationListView and the applicationView) */
-    var ApplicationListView = Vue.extend({});
+    var ApplicationListView = Vue.extend({
+        template:
```

Comment:
That's what I did first, but it turned out that it's only available with ES6. So if you don't mind we'll do that later when using Webpack with browserify and babelify.